webclient/heslar/signals.py

```diff
@@ -89,6 +90,8 @@ def save_metadata_heslar_hierarchie(sender, instance: HeslarHierarchie, created,
     Funkce pro uložení metadat heslář - hierarchie.
     """
     logger.debug("heslo.signals.save_metadata_heslar_hierarchie.start")
+    invalidate_model(HeslarHierarchie)
+    invalidate_model(Heslar)
```

Comment:
@pesikj Tohle si nejsem jistý, že pomůže - protože ono se to nikde nedostane ani k tomu `heslo.signals.save_metadata_heslar_hierarchie.start`. Je to dobře vidět tady z logů:
-- na testu vidím danou položku v logu: http://192.168.254.30:5601/app/discover#/?_g=(filters:!(),query:(language:kuery,query:''),refreshInterval:(pause:!f,value:5000),time:(from:'2024-08-03T08:39:40.317Z',to:'2024-08-03T10:39:47.697Z'))&_a=(columns:!(message),filters:!(),index:'44e15cc6-98d1-4cc3-815f-27089e09e9d1',interval:auto,query:(language:kuery,query:heslo.signals.save_metadata_heslar_hierarchie.start),sort:!(!('@timestamp',desc)))
- na TR žádná není, i když jsem tam danou operaci také dělal: http://192.168.254.34:5601/app/discover#/?_g=(filters:!(),query:(language:kuery,query:''),refreshInterval:(pause:!f,value:5000),time:(from:'2024-08-03T08:39:40.317Z',to:'2024-08-03T10:39:47.697Z'))&_a=(columns:!(message),filters:!(),hideChart:!f,index:'858d02c4-091e-44c1-9600-ddb1a3dbaf32',interval:auto,query:(language:kuery,query:heslo.signals.save_metadata_heslar_hierarchie.start),sort:!(!('@timestamp',desc)))